### PR TITLE
update using pajarito section of readme

### PR DESCRIPTION
**Pull request by mlubin**
Preview: https://github.com/mlubin/Pajarito.jl/blob/ml/readme2/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,19 @@ Pajarito is a solver for mixed-integer convex optimization problems. These are o
...
+Pajarito has two entirely separate algorithms depending on the form in which the input is provided. The first is the **derivative-based nonlinear** algorithm, where the approach used is analogous to that of [Bonmin](https://projects.coin-or.org/Bonmin) and will perform similarly, with the primary advantage of being able to easily swap-in various mixed-integer and convex subproblem solvers which Bonmin does not support. The second is **conic** algorithm which is the new approach proposed in the publications describing Pajarito. The conic algorithm currently supports second-order cones, exponential cones, and positive semidefinte cones. A problem provided in conic form may solve faster than an identical problem encoded in derivative-based nonlinear form because conic form naturally encodes extended formulations; however, [Hijazi et al.](http://www.optimization-online.org/DB_FILE/2011/06/3050.pdf) suggest manual reformulation techniques which achieve many of the algorithmic benefits of conic form.
```
Comment (joehuchette) [inline README.md]: *semidefinite

**Comment by chriscoey** [comment]
I will update options after you merge.

One thing I would like to see changed is the "subproblem solvers" section. I think the terminology should be MIP solver AKA "OA solver" and continuous solver AKA "subproblem solver". We refer to cuts derived from the continuous subproblem as "subproblem cuts" so I think this is just overall clearer.

File: README.md (hunk 1 above)
Comment (mlubin) [inline README.md]: fixed

**Comment by chriscoey** [comment]
Further, I don't see the need to put both types of solver under the same section heading. Have a section for MIP/OA solver (which may be MILP or MISOCP) and one for continuous/subproblem solver (which may be conic or NLP).

**Comment by mlubin** [comment]
Right now I'm just trying to correct the misleading information that's currently in the readme, but we should certainly write more about the subproblem solvers and how to properly use them (including which options need to be set).

**Comment by mlubin** [comment]
CC @ccoffrin

**Pull request merged**